### DISCUSSION TITLE
iPadOS cursor hovering over “Add Favorites” button

### DISCUSSION THREAD

**Discussion by AustinConlon** [community/community#11074] [Mobile] [answered]
This looks like unexpected behavior, notice the background color doesn't change during hover state and it looks like the cursor just disappears. For comparison, I hover over the "..." button in the same screen recording which gives a focus appearance I expected.

https://user-images.githubusercontent.com/1305957/152660446-1b32833f-ebcd-4a6b-bbf3-fb075aa4b577.MOV

**Comment by brianlovin** [comment, marked as answer]
Hey @AustinConlon thanks for the note — right now the button is getting a hover state with motion + shadow, are you expecting a different kind of indicator that the cursor is on the button?
